### PR TITLE
[BACKLOG-15219][BACKLOG-15221][BACKLOG-15220] Configuration fixes

### PR DESCRIPTION
**Pull request by nantunes**
@pentaho/millenniumfalcon please review.

Merge with https://github.com/pentaho/pentaho-det/pull/718.

### CODE REVIEW

**Comment by nantunes** [comment]
@diogofscmariano check if this affects your work.

**Comment by diogofscmariano** [comment]
yes it does, need to change cgg. Thanks for the heads up!!!!!!

**Comment by diogofscmariano** [comment]
please merge this pr along with this work
https://github.com/webdetails/cgg/pull/94

File: impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js [hunk 1]
```diff
@@ -38,6 +38,42 @@ define([
...
+        select: {
+          priority: -1,
+          type: [
+            "pentaho/visual/models/heatGrid"
+          ]
+        },
+        apply: {
+          props: {
+            colorSet: {
+              value: "blue"
```
Comment (dcleao) [inline impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js]: Shouldn't this `colorSet` default value  be accompanied with changes to the "blue" color scales? See: https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/master/impl/client/src/main/javascript/web/pentaho/visual/color/paletteRegistry.js#L288

Comment (nantunes) [inline impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js]: For now it is ok not changing the palette. UX review decide what to do.

**Pull request merged**